nix/pkgs/btrfs-progs/default.nix

```diff
@@ -0,0 +1,63 @@
+# Source: https://github.com/NixOS/nixpkgs/blob/b483ef3cc866ae6afed65115ba6f0c6b19efce49/pkgs/tools/filesystems/btrfs-progs/default.nix
+{ lib
+, stdenv
+, fetchurl
+, pkg-config
+, attr
+, acl
+, zlib
+, libuuid
+, e2fsprogs
+, lzo
+, asciidoc
+, xmlto
+, docbook_xml_dtd_45
+, docbook_xsl
+, libxslt
+, zstd
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "btrfs-progs";
+  version = "5.14.1";
```

Comment:
Can we not pin the version? Also we use `5.6.x` for btrfs operations on csi, so maybe it should be at least be in par with that.